### PR TITLE
Core 148 fix

### PR DESCRIPTION
**Pull request by BoulderBrains**
During QA of the repayment schedule Lauren happened to look at a "Funded" loan in development while being logged out, in that situation she was getting this result, as the check to show the repayment schedule was only looking at the loans "anonymizationLevel".
[Loan](https://www.dev.kiva.org/lend-beta/2293062) 
Screen Shot 2022-01-12 at 9 43 35 AM
So I was showing the repayment schedule but no data was coming through for it.  

I compared the older borrower profile, logged in and logged out states and noticed that for funded loans we were only showing the repayment schedule while being logged in. 

The check I've now put in place: 
```
displayRepaymentSchedule() {
	// check anonymization of loan, if it's not set to
	// full anonymization, continue
	if (this.loan.anonymizationLevel !== 'full') {
		// if loan is fundraising, always display the repayment schedule
		if (this.loan.status === 'fundraising') {
			return true;
		}
		// otherwise look at the isSupporter flag to determine if
		// the repayment schedule should be shown to current user,
		// if they are logged in
		return this.isSupporter;
	}
	// if loan is set to full anonymization, return false
	// because the repayment schedule should not be shown.
	return false;
}
```

Now we're only showing the repayment schedule: 
- if the loan.anonymizationLevel !== 'full'
& if the loan is fundrasing
& if the loan is not fundraising, if the user is a supporter of that loan

### CODE REVIEW

File: src/components/BorrowerProfile/DetailsTabs.vue [hunk 1]
```diff
@@ -197,6 +197,26 @@ export default {
 		trusteeTabId() {
 			return `tab-panel-${this.name}-trustee`;
 		},
+		isSupporter() {
+			return this.loan.lentTo;
```
Comment (mcstover) [inline src/components/BorrowerProfile/DetailsTabs.vue:201]: prob want too use optional chaining on `loan` here.

Comment (mcstover) [reply src/components/BorrowerProfile/DetailsTabs.vue:201]: Same below

Comment (BoulderBrains) [reply src/components/BorrowerProfile/DetailsTabs.vue]: @mcstover Below are checks looking at the value and not assigning it. It was my understanding that optional chaining is only needed when making an assignment to a const. Is that wrong?

Comment (mcstover) [reply src/components/BorrowerProfile/DetailsTabs.vue]: Yep, that's good. Please add your "[nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)" operator so this returns `false` if it fails.

**Review by mcstover — APPROVED** [review]

**Review by ryan-ludwig — APPROVED** [review]

**Pull request merged**